kaori/plugins/gacha/tui.py

```diff
@@ -268,7 +296,7 @@ def battle_blocks(attacker: Card, defender: Card, battle_url: str):
                 "type": "button",
                 "text": {
                     "type": "plain_text",
-                    "text": "Watch Battle",
+                    "text": "Start Battle",
```

Comment:
I think you changed this in other PR

Reply:
ffs I cannot do merges on my laptop screen

```suggestion
                    "text": "Watch Battle",
```